notebooks/data-sources/Bugzilla/bugzilla_EDA.ipynb

```diff
@@ -0,0 +1,3552 @@
+       "      <td>1</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>[{'ext_description': 'None', 'ext_bz_id': 139, 'ext_priority': 'None', 'bug_id': 1887432, 'ext_bz_bug_id': 'RHBA-2020:4270', 'id': 1638493, 'ext_status': 'None', 'type': {'must_send': False, 'can_send': False, 'description': 'Red Hat Product Errata', 'can_get': False, 'url': 'https://access.redhat.com/errata/', 'id': 139, 'send_once': False, 'type': 'None', 'full_url': 'https://access.redhat.com/errata/%id%'}}]</th>\n",
+       "      <td>1</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>[{'ext_description': 'Bug 1929944: etcdInsufficientMembers is wrong when etcd is in a pod', 'ext_bz_id': 131, 'ext_priority': 'None', 'bug_id': 1929944, 'ext_bz_bug_id': 'openshift/cluster-monitoring-operator/pull/1064', 'id': 1748979, 'ext_status': 'open', 'type': {'must_send': False, 'can_send': False, 'description': 'Github', 'can_get': True, 'url': 'https://github.com/', 'id': 131, 'send_once': False, 'type': 'GitHub', 'full_url': 'https://github.com/%id%'}}]</th>\n",
+       "      <td>1</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "                                                    external_bugs\n",
+       "[]                                                           1243\n",
+       "[{'ext_description': 'CVO not using proxy setti...              1\n",
+       "[{'ext_description': 'Bug 1899725: Reduce Quick...              1\n",
+       "[{'ext_description': 'None', 'ext_bz_id': 139, ...              1\n",
+       "[{'ext_description': 'Bug 1929944: etcdInsuffic...              1"
+      ]
+     },
+     "execution_count": 72,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "bugs_df[\"external_bugs\"].value_counts().to_frame().head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8516f0f6",
+   "metadata": {},
+   "source": [
+    "### platform"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 73,
+   "id": "407494c8",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "0    Unspecified\n",
+       "1    Unspecified\n",
+       "2         x86_64\n",
+       "3    Unspecified\n",
+       "4            All\n",
+       "Name: platform, dtype: object"
+      ]
+     },
+     "execution_count": 73,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "bugs_df[\"platform\"].head()"
```

Comment:
For the next few metrics you print out a *.head() and *.unique(). Seems a little redundant. I would remove them both and just display the *value_counts().

Reply:
+1, makes sense :)